SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -562,6 +562,20 @@ - (void)handleProtocolEndServiceNAKMessage:(SDLProtocolMessage *)endServiceNAK {
     [self sdl_transitionToStoppedState:endServiceNAK.header.serviceType];
 }
 
+- (void)handleProtocolEndService:(SDLProtocolMessage *)endService forSession:(Byte)session {
+    // Respond with EndService ACK to address SYNC bug
```

Comment:
Check here if the service type is video. Otherwise we will respond to both video and audio end session control frames.  

```
if (endService.header.serviceType != SDLServiceTypeVideo) { return; }
```

Reply:
Fixed in most recent commit.